src/pages/collection/series/SeriesOverview.tsx

```diff
@@ -31,9 +31,17 @@ const SeriesOverview = () => {
   const { seriesId } = useParams();
 
   const seriesQuery = useSeriesQuery(toNumber(seriesId!), { includeDataFrom: ['AniDB', 'TMDB'] }, !!seriesId);
-  const series = useMemo(() => seriesQuery?.data ?? {} as SeriesType, [seriesQuery.data]);
+  const series = useMemo(
+    () =>
+      seriesQuery?.data
+        ?? {
```

Comment:
In case someone reading this is wondering, this will prevent any errors where we try to access an undefined property as if it were defined and make the runtime throw as a result.